lib/ansible/modules/monitoring/zabbix/zabbix_user.py

```diff
@@ -0,0 +1,621 @@
+                                                   disaster=dict(type='bool', default=True)),
+                                               default=dict(
+                                                   not_classified=True,
+                                                   information=True,
+                                                   warning=True,
+                                                   average=True,
+                                                   high=True,
+                                                   disaster=True
+                                               )),
+                                 active=dict(type='bool', default=True)
+                             )),
+            type=dict(type='str', default='Zabbix user', choices=['Zabbix user', 'Zabbix admin', 'Zabbix super admin']),
+            state=dict(type='str', default="present", choices=['present', 'absent']),
+            timeout=dict(type='int', default=10)
+        ),
+        supports_check_mode=True
+    )
+
+    if not HAS_ZABBIX_API:
+        module.fail_json(msg=missing_required_lib('zabbix-api', url='https://pypi.org/project/zabbix-api/'),
+                         exception=ZBX_IMP_ERR)
+
+    server_url = module.params['server_url']
+    login_user = module.params['login_user']
+    login_password = module.params['login_password']
+    http_login_user = module.params['http_login_user']
+    http_login_password = module.params['http_login_password']
+    validate_certs = module.params['validate_certs']
+    alias = module.params['alias']
+    name = module.params['name']
+    surname = module.params['surname']
+    usrgrps = module.params['usrgrps']
+    passwd = module.params['passwd']
+    override_passwd = module.params['override_passwd']
+    lang = module.params['lang']
+    theme = module.params['theme']
+    autologin = module.params['autologin']
+    autologout = module.params['autologout']
+    refresh = module.params['refresh']
+    rows_per_page = module.params['rows_per_page']
+    after_login_url = module.params['after_login_url']
+    user_medias = module.params['user_medias']
+    user_type = module.params['type']
+    state = module.params['state']
+    timeout = module.params['timeout']
+
+    if autologin:
+        autologin = '1'
+    else:
+        autologin = '0'
+
+    user_type_dict = {
+        'Zabbix user': '1',
+        'Zabbix admin': '2',
+        'Zabbix super admin': '3'
+    }
+    user_type = user_type_dict[user_type]
+
+    zbx = None
+
+    # login to zabbix
+    try:
+        zbx = ZabbixAPI(server_url, timeout=timeout, user=http_login_user, passwd=http_login_password,
+                        validate_certs=validate_certs)
+        zbx.login(login_user, login_password)
+    except Exception as e:
+        module.fail_json(msg="Failed to connect to Zabbix server: %s" % e)
+
+    user = User(module, zbx)
+
+    user_ids = {}
+    zbx_api_version_split = zbx.api_version().split('.')
```

Comment:
`zabbix_template` uses [LooseVersion](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/monitoring/zabbix/zabbix_template.py#L351) for version comparison, can you unite it please?

Reply:
Fix to be able to compare versions using `LooseVersion`.